fe/src/main/jflex/sql_scanner.flex

```diff
@@ -352,7 +352,7 @@ import org.apache.doris.common.util.SqlUtils;
         keywordMap.put("with", new Integer(SqlParserSymbols.KW_WITH));
         keywordMap.put("work", new Integer(SqlParserSymbols.KW_WORK));
         keywordMap.put("write", new Integer(SqlParserSymbols.KW_WRITE));
-        keywordMap.put("||", new Integer(SqlParserSymbols.KW_OR));
```

Comment:
If you delete the KW_OR, the sql parser will thrown exception which is include lots of KW_OR.
Maybe you should not change the name of KW_OR

Reply:
This is a change for KW_OR.
There is two symbols map to KW_OR at the same time in origin:
keywordMap.put("or", new Integer(SqlParserSymbols.KW_OR));
keywordMap.put("||", new Integer(SqlParserSymbols.KW_OR));
So this commit only change '||' and map it to a new id 'KW_PIPE'